test/e2e/consoleplugin/smoke_test/consoleplugin_test.go

```diff
@@ -76,5 +79,22 @@ var _ = Describe("[ConsolePlugin]", func() {
 		runtime.Initialize(cp, cl.Namespace, "logging-view-plugin")
 		Eventually(func() error { return c.Get(cp) }, time.Minute, time.Second).Should(Succeed())
 		By("Got console plugin: " + test.JSONLine(cp))
+
+		Eventually(func() error {
+			pods := &corev1.PodList{}
+			selector := client.MatchingLabels{constants.LabelK8sName: console.Name}
+			if err := c.List(pods, selector, client.InNamespace(cl.GetNamespace())); err != nil {
+				return err
+			}
+			for _, pod := range pods.Items {
+				for _, cond := range pod.Status.Conditions {
+					if cond.Type == corev1.PodReady && cond.Status == corev1.ConditionTrue {
+						return nil
+					}
+				}
+			}
+			return fmt.Errorf("No pods found in ready condition")
+		},
```

Comment:
Is indentation off here?

Reply:
:man_shrugging:  I ran it through "make fmt"